src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -53,7 +55,8 @@
  *
  */
 public abstract class CommandLineProgram implements CommandLinePluginProvider {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+
+    protected final Logger logger = LogManager.getLogger(CommandLineProgram.class);
```

Comment:
Why change this?  If we change it this way it should probably be made static and private.